build/IceRpc.Version.props

```diff
@@ -5,7 +5,8 @@
         <!-- The Protobuf version used by IceRpc.Protobuf -->
         <ProtobufVersion>26.1</ProtobufVersion>
         <NuGetProtobufVersion>3.$(ProtobufVersion)</NuGetProtobufVersion>
-        <TargetFramework>net8.0</TargetFramework>
-        <NetRuntimeVersion>8.0.*</NetRuntimeVersion>
+        <!-- The target framework used by assemblies -->
+        <IceRpcTargetFramework>net8.0</IceRpcTargetFramework>
```

Comment:
Rather than setting the TargetFramework directly here, we use `IceRpcTargetFramework` variable, this allows importing the files into other projects without changing the TargetFramework.